resources/user-agents/browsers/android-webview/android-webview-1-0.json

```diff
@@ -493,7 +493,7 @@
             "HTC_DesireS": "HTC Desire S",
             "HTC_Sensation": "HTC Z710",
             "HTC_Sensation_Z710e": "HTC Z710e",
-            "HTC_SensationXE_Beats": "HTC Z710",
```

Comment:
The Z710 is just "HTC Sensation", so doesn't seem like it should be the device used for this token.